src/commonTest/kotlin/exchange.dydx.abacus/payload/TriggerOrdersInputTests.kt

```diff
@@ -20,6 +21,53 @@ class TriggerOrderInputTests : V4BaseTests() {
     fun testInputs() {
         setup()
 
+        // Tests on ETH-USD market with
+        // - notional total: 1753.2932
+        // - size: 0.5
+        // - leverage: 2.054737105604498
+
+        val leverage = 2.054737105604498;
+
+        testDefaults()
+        testSetPositionSize()
+        testStopLossInput(leverage)
+        testTakeProfitInput(leverage)
+    }
+
+    @Test
+    fun testLessThanZeroLeverageInputs() {
+        setup()
+
+        // Tests on ETH-USD market with
+        // - notional total: 1753.2932
+        // - size: 0.5
+        // - leverage < 1 (defaults to 1 in calculations)
+
+        test(
+            {
+                perp.rest(
+                    AbUrl.fromString("$testRestUrl/v4/addresses/cosmo"),
+                    mock.accountsChannel.v4_accounts_received_for_calculation_2,
+                    0,
+                    null,
+                )
+            },
+            null,
+        )
+
+        val leverage = 1.0;
```

Comment:
shouldnt this be less than 1?

Reply:
oh but for the calculations, they should be 1 (and I'm just using this variable for calculations)